.github/workflows/1-experiment.yaml

```diff
@@ -1,85 +1,74 @@
-name: train-model
-
+name: Train
 on:
   push:
-    branches-ignore: # exclude master and dev below
-      - main
-      - dev
-
+    # don't run on `main` and `dev` commits
+    branches-ignore: [main, dev]
 jobs:
   deploy-runner:
-    if: contains(github.event.head_commit.message, '[cml]')
     runs-on: ubuntu-latest
+    environment: cloud
+    permissions:
+      contents: read
+      id-token: write
     steps:
       - uses: actions/checkout@v2
       - uses: iterative/setup-cml@v1
+      - uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-region: us-east-1
+          role-to-assume: arn:aws:iam::342840881361:role/SandboxUser
```

Comment:
One problem with this approach (while it may work well for us and be more "correct") is it's probably not easy for beginners to replicate. The `env.AWS_` stuff is probably easier.

WDYT @iterative/cml?

Reply:
Yeah, I think we can keep this as a showcase example + add notes in the README on what things to change if people want to run this with their personal AWS creds.

Reply:
(Or better yet — shooting myself in both feet — providing Terraform configuration to use OpenID Connect on every supported cloud and continuous integration system)